index.html

```diff
@@ -1658,6 +1680,219 @@ <h4 id="example-get-all-patient-types">Example: Delete Payment Profile Response<
             </pre>
             <p>&nbsp;</p>
 
+            <!-- Webhooks -->
+            <h1 id='webhooks'>Webhook Events</h1>
+            <p>You can choose to receive event notifications known as "Webhooks" when certain events happen related to payments you have created. For example, 
+                when you create a payment using this API, if that payment is refunded through another system, a webhook can be sent to a URL that you specify
+                when that refund happens. </p>
+            <p>Webhook events are JSON objects that are sent using a POST method.</p>
+            <h2 id="webhooks-authentication">Authentication</h2>
+            <p>When you receive a webhook event, you can verify the request by using a standard RSA sign/verify process. Each request contains a header 
+                parameter "webhook-signature". Below are example steps to take to verify the request.
+
+            </p>
+            <ol>
+                <li>Get the <code>webhook-signature</code> value from the header of the incoming request. It is a base64 encoded string.</li>
+                <li>Get our public key by calling the <code>/api/public/Access/WebhookVerification</code> endpoint described below. You should cache this value locally.</li>
+                <li>Use the security key to verify the signature.</li>
+            </ol>
+            <p>You can <a href="https://en.wikipedia.org/wiki/RSA_(cryptosystem)" target="_blank">learn more about RSA here</a>.</p>
+            <p>Here is a <a href="https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsacryptoserviceprovider.verifydata?view=netframework-4.8" target="_blank">C# RSA sign/verify example</a>.</p>
+
+
+            <h4 id="get-public-key-endpoint">Get Public Key Endpoint</h4>
+            <p>
+                <code>
+                    GET http://payfac.api.nextech.com/api/public/Access/WebhookVerification
+                </code>
+            </p>
+            <h4>Response</h4>
+            <p id="get-public-key-response">
+                <a href="#key-vault-response">KeyVaultResponse</a>
+            </p>
+
+            <h4>Example Response</h4>
+            <pre class="center-column">
+{
+    "N": "oSavKivsDmHseYWFu8zH............1ygwWf9aFy/kuDLxVIzRrRnyKW3U",
+    "E": "AQAB",
+    "Algorithm": "RS256"
+    "KeySize": "2048"
```

Comment:
KeySize is Int, not sure this comes across with "" - it's in a JSON object. Could we verify this?

Reply:
updated